src/lib/viewers/__tests__/BaseViewer-test.js

```diff
@@ -1296,6 +1300,44 @@ describe('lib/viewers/BaseViewer', () => {
         });
     });
 
+    describe('handleAnnotationsOnLoad()', () => {
```

Comment:
Add test for if specified active annotation id is not found in the list of annotations?